deltachat-ios/DC/DcMsg+Extension.swift

```diff
@@ -48,13 +48,18 @@ extension DcMsg: MessageType {
     }
 
     internal func createVideoMessage(text: String) -> MessageKind {
-        let thumbnail = DcUtils.generateThumbnailFromVideo(url: fileURL)
         if text.isEmpty {
+            var thumbnail: UIImage?
+            if let fileURL = fileURL {
+                thumbnail = ThumbnailCache.shared.restoreImage(key: fileURL.absoluteString)
+            }
             return MessageKind.video(Media(url: fileURL, image: thumbnail))
         }
-        let attributedString = NSAttributedString(string: text, attributes: [NSAttributedString.Key.font: UIFont.preferredFont(forTextStyle: .body),
-                                                                             NSAttributedString.Key.foregroundColor: DcColors.defaultTextColor])
-        return MessageKind.videoText(Media(url: fileURL, image: thumbnail, text: [attributedString]))
+        let attributedString = NSAttributedString(string: text, attributes: [
+            NSAttributedString.Key.font: UIFont.preferredFont(forTextStyle: .body),
+            NSAttributedString.Key.foregroundColor: DcColors.defaultTextColor]
+        )
+        return MessageKind.videoText(Media(url: fileURL, text: [attributedString]))
```

Comment:
There is a  🐞 
Cells with combined text and video should also provided with the thumbnail here. (`Media(url: fileURL, image: thumbnail, text: [attributedString])`)
TextMediaMessageCells should also handle the situation if there's no thumbnail yet there.

Reply:
Tx! Fixed